addons/point_of_sale/static/src/js/gui.js

```diff
@@ -184,20 +184,29 @@ var Gui = core.Class.extend({
     close_other_tabs: function() {
         var self = this;
 
+        // avoid IE to close itself
+        var now = +new Date();
```

Comment:
In fact, regardless of ```Math.random()``` or ```Date.now()``` I think we should call the field ```window_uid``` instead of ```time```. It better reflects the meaning and further use of that variable.